csharp/ql/src/experimental/Security Features/JsonWebTokenHandler/delegated-security-validations-always-return-true.ql

```diff
@@ -0,0 +1,24 @@
+/**
+ * @name Delegated security sensitive validations for JsonWebTokenHandler always return true, medium precision
+ * @description Security sensitive validations for `JsonWebTokenHandler` are being delegated to a function that seems to always return true.
+ *   Higher precision version checks for exception throws, so less false positives are expected.
+ * @kind problem
+ * @tags security
+ *       JsonWebTokenHandler
+ *       manual-verification-required
+ * @id cs/JsonWebTokenHandler/delegated-security-validations-always-return-true
+ * @problem.severity error
+ * @precision high
+ */
+
+import csharp
+import DataFlow
+import JsonWebTokenHandlerLib
+
+from
+  TokenValidationParametersPropertyWriteToValidationDelegated tv, Assignment a,
+  CallableAlwaysReturnsTrueHigherPrecision e
+where a.getLValue() = tv and a.getRValue().getAChild*() = e
```

Comment:
Is`a.getRValue().getAChild*` correct? In the below case, the check would report the property write, although property is not assigned to a lambda that always returns true.
```
prop = condition 
  ? (...) => true 
  : (...) => false;
```